config/devel/rbac/definitions.json

```diff
@@ -48,7 +48,8 @@
       "name": "Compliance viewer",
       "description": "A Compliance role that grants read access to any Compliance resource.",
       "system": true,
-      "version": 2,
+      "platform_default": true,
```

Comment:
What's the difference here between `admin_default` and `platform_default`?

Reply:
I can't think of how they would matter in the development setup. We could probably just use a single role with Compliance Administrator permissions.

I've added other roles and their defaults to match configuration changes in production.